app/assets/javascripts/Components/starter_file_manager.jsx

```diff
@@ -242,9 +242,11 @@ class StarterFileManager extends React.Component {
                 canFilter={false}
               />
               <button
+                aria-label="sfg-action-button"
```

Comment:
`aria-label` is meant for human-readable strings, so it would be good to use an internationalized string throughout there, and then in the tests